broadinstitute/single_cell_portal_core
# Report client-side upload validation errors in SCP UI (SCP-3546)

This implements a basic component to show validation errors after files are selected, but before upload.  It matures the bare native alert in #1116 into a Bootstrap alert rendered in React.

https://user-images.githubusercontent.com/1334561/129970256-081cfc70-b107-4a3b-b21a-75c18caf0a19.mov

Future work includes validating more types of errors, and logging them to Mixpanel.

To test:
* Do test steps from #1116 
* Verify that styled notice appears instead ofbare native alert

This satisfies SCP-3546.
